src/main/java/org/jabref/gui/customentrytypes/StandardEntryTypeViewModel.java

```diff
@@ -0,0 +1,9 @@
+package org.jabref.gui.customentrytypes;
+
+import org.jabref.model.entry.BibEntryType;
+
+public class StandardEntryTypeViewModel extends EntryTypeViewModel {
```

Comment:
Is this class really necessary? Wouldn't it be enough to make EntryTypeViewModel not abstract and to extend it to the CustomEntryTypeViewModel?

Reply:
Yeah, I also though of that. I can change it.